### PR TITLE
html css popup

### PR DESCRIPTION
**Pull request by ustymchyk**
### HTML & CSS practice

[Demo](https://ustymchyk.github.io/html-css-popup/) |
[Code base](https://github.com/ustymchyk/html-css-popup)

The code is submitted in a dedicated feature branch.
Please, review.

### CODE REVIEW

File: submissions/ustymchyk/html-css-popup/index.html [hunk 1]
```diff
@@ -0,0 +1,151 @@
...
+        </div>
+      </div>
+      <a class="header__item notifications" href>Notifications menu</a>
+      <a class="header__item user" href>Account menu</a>
+    </nav>
+  </header>
+  <main></main>
+</body>
+
+</html>
```
Comment (miroslav-kolomiets) [inline submissions/ustymchyk/html-css-popup/index.html:151]: Add new line.

File: submissions/ustymchyk/html-css-popup/index.html [hunk 2]
```diff
@@ -0,0 +1,151 @@
...
+  <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/meyer-reset/2.0/reset.min.css">
+  <link rel="stylesheet" href="index.css">
+</head>
+
+<body>
+  <header class="header">
+    <nav class="nav">
+      <a class="header__item" href>Mail</a>
+      <a class="header__item" href>Images</a>
+      <input type="checkbox" id="menu" class="header__item menu-btn">
```
Comment (miroslav-kolomiets) [inline submissions/ustymchyk/html-css-popup/index.html:17]: Make your menu with the same structure, for example:
```suggestion
     <ul>
        <li class="nav__item">
          <a class="header__item" href>Mail</a>
        </li>
        <li class="nav__item">
          <a class="header__item" href>Images</a>
        </li>
        <li class="nav__item">
          <input type="checkbox" id="menu" class="header__item menu-btn">
        </li>
      </ul>
```

File: submissions/ustymchyk/html-css-popup/index.html [hunk 3]
```diff
@@ -0,0 +1,151 @@
...
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>HTML and CSS popup</title>
+  <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/meyer-reset/2.0/reset.min.css">
+  <link rel="stylesheet" href="index.css">
+</head>
+
+<body>
+  <header class="header">
+    <nav class="nav">
+      <a class="header__item" href>Mail</a>
```
Comment (miroslav-kolomiets) [inline submissions/ustymchyk/html-css-popup/index.html:15]: Wrong class name, it's part of navigation, so it's better name it `nav__item`.

**Review by miroslav-kolomiets — CHANGES_REQUESTED** [review]
Nice job! But something can be improved.

File: submissions/ustymchyk/html-css-popup/index.html (hunk 2 above)
Comment (ustymchyk) [reply submissions/ustymchyk/html-css-popup/index.html:17]: I thought it is not allowed to use a **div** inside a **li**. Thanks.

File: submissions/ustymchyk/html-css-popup/index.html (hunk 3 above)
Comment (ustymchyk) [reply submissions/ustymchyk/html-css-popup/index.html:15]: Agree, thanks.

File: submissions/ustymchyk/html-css-popup/index.html (hunk 2 above)
Comment (miroslav-kolomiets) [reply submissions/ustymchyk/html-css-popup/index.html]: It is valid to use a `div` inside a `li`, but in my example there is no such situation.

Comment (ustymchyk) [reply submissions/ustymchyk/html-css-popup/index.html]: In this situation i have to add .menu after input. In other case i can`t user :checked + .menu selector to open/hide menu. So, div with menu will be inside li.

Comment (ustymchyk) [reply submissions/ustymchyk/html-css-popup/index.html]: image

**Review by miroslav-kolomiets — CHANGES_REQUESTED** [review]
It seems you forgot to update github pages
Снимок экрана 2020-11-16 в 11 06 51 PM

**Comment by ustymchyk** [comment]
> It seems you forgot to update github pages
> Снимок экрана 2020-11-16 в 11 06 51 PM

done

**Comment by A-Ostrovnyy** [comment]
@ustymchyk 
For future PR's use our template for the 1-st message:
```
### Task name

[Demo](link to demo) |
[Code base](link to code base)

The code is submitted in a dedicated feature branch.
Please, review.
```

**Comment by ustymchyk** [comment]
@A-Ostrovnyy 
I'm understanding this correctly?
image

**Comment by A-Ostrovnyy** [comment]
@ustymchyk yes, you understand correctly.

**Comment by ustymchyk** [comment]
So, time to accept my PR?)

**Review by A-Ostrovnyy — APPROVED** [review]

**Comment by A-Ostrovnyy** [comment]
@miroslav-kolomiets your turn)

**Comment by ustymchyk** [comment]
@miroslav-kolomiets review please

**Review by miroslav-kolomiets — APPROVED** [review]
Great work!

**Pull request merged**